test/InputOutput/hybrid3dcubedsphere.jl

```diff
@@ -10,6 +10,18 @@ using ClimaCore:
     DataLayouts,
     InputOutput
 
+using ClimaComms
+usempi = get(ENV, "CLIMACORE_DISTRIBUTED", "") == "MPI"
+if usempi
+    using ClimaCommsMPI, MPI
+    const comms_ctx = ClimaCommsMPI.MPICommsContext()
+    ClimaComms.init(comms_ctx)
+    filename = MPI.bcast(tempname(), 0, MPI.COMM_WORLD)
```

Comment:
Can we add a quick status printout indicating that this is a distrubuted run with $nprocs processes!